.github/workflows/website-schema.yml

```diff
@@ -0,0 +1,21 @@
+name: "Website config validation"
+
+on:
+  workflow_call:
+    inputs:
+      json-file:
```

Comment:
Do we have repositories where the path is not the default one? If not, YAGNI.

Reply:
It is Yagni (It's a draft after all). I did some testing on private repositories where I used different files and filepaths. It will be removed once it becomes a regular Pull Request.